packages/dao/contracts/Api3Template.sol

```diff
@@ -1,9 +1,13 @@
 //SPDX-License-Identifier: MIT
 pragma solidity 0.4.24;
 
-import "./Api3BaseTemplate.sol";
+import "@aragon/templates-shared/contracts/BaseTemplate.sol";
+import "./utils/Api3Voting.sol";
```

Comment:
Instead of duplicating Api3Voting.sol, @api3-dao/api3-voting should export the contract and @api3-dao/dao should import it (and there is a similar issue with how @api3-dao/api3-voting copy pastes the pool artifact for tests). I can open a quick PR for that once you merge this.

Reply:
> Instead of duplicating Api3Voting.sol, @api3-dao/api3-voting should export the contract and @api3-dao/dao should import it (and there is a similar issue with how @api3-dao/api3-voting copy pastes the pool artifact for tests). I can open a quick PR for that once you merge this.

that would be awesome! Thanks